src/ui/styles/_cards.css

```
@@ -0,0 +1,196 @@
+.card-region__body {
+    padding: 8px 10px 10px;
+    border-top: 1px solid var(--c-border);
+}
+
+.card-region--unresolved {
+    border-color: var(--c-danger-40);
+}
+
+.card-row .account-row__badge {
+    --card-tier-color: var(--c-text-muted);
+    display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)